developer-docs-site/docs/tutorials/your-first-nft-sdk.md

```diff
@@ -0,0 +1,376 @@
+```
+
+`common.py` initializes these values as such:
+```python
+:!: static/sdks/python/examples/common.py section_1
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+:::tip
+
+The URLs for both services, by default, point to our devnet services, however, they can be configured via setting following environmental variables: `APTOS_NOD
+E_URL` and `APTOS_FAUCET_URL`.
+
+:::
+
+
+### Step 4.2: Creating local accounts
+
+The next step is to create two accounts from the locally. [Accounts][account_basics] represent both on and off-chain state. The off-chain state consists of an
+address and the public, private key pair used to authenticate ownership. This step demonstrates how to generate that off-chain state.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```python
+:!: static/sdks/python/examples/simple-nft.py section_2
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+### Step 4.3: Creating blockchain accounts
+
+In Aptos, each account must have an on-chain representation in order to support receive tokens, coins, and interacting in other DApps. In Aptos, an account rep
+resents a medium for storing assets, hence it must be explicitly created. This example leverages the Faucet to create and fund Alice's account and to just crea
+te Bob's:
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```python
+:!: static/sdks/python/examples/simple-nft.py section_3
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+### Step 4.4: Creating a collection
+
+Now begins the process of creating tokens. First, the creator must create a collection to store the tokens in. A collection can contain 0, 1, or many distinct tokens within it. The collection does not restrict the attributes of the tokens, it is only a container.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+Your application will call `create_collection`:
+```python
+:!: static/sdks/python/examples/simple-nft.py section_4
+```
+
+`create_collection`'s API, which returns a transaction hash:
+```python
+:!: static/sdks/python/aptos_sdk/client.py create_collection
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+### Step 4.5: Creating a token
+
+To create a token, the creator must specify an associated collection. A token must be associated with a collection and that collection must have remaining token count that can be minted. There are a lot of attributes associated with a token, but the helper API only exposes the minimal amount required to create static content.
```

Comment:
`must have a remaining token count` missing an article

`There are many of attributes` many preferred over a lot